pkg/operator/configobservation/configobservercontroller/observe_config_controller.go

```diff
@@ -1,6 +1,7 @@
 package configobservercontroller
 
 import (
+	"github.com/openshift/cluster-openshift-controller-manager-operator/pkg/operator/configobservation/controllers"
```

Comment:
nit: This should be at the end of import

Reply:
Done.